src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -22,9 +22,10 @@
 (defn get-actions
   [{:keys [outgoing content pinned outgoing-status deleted? deleted-for-me?] :as message-data}
    {:keys [edit-enabled show-input? community? can-delete-message-for-everyone?
-           message-pin-enabled group-chat group-admin?]}]
+           message-pin-enabled group-chat group-admin?]}
+   content-type]
   (concat
-   (when (and outgoing edit-enabled (not (or deleted? deleted-for-me?)))
```

Comment:
do we really need to remove this part of condition `(not (or deleted? deleted-for-me?))`? Is it related to this PR?

Reply:
Hmm I'll check and let you know, maybe I messed it up when rebasing, thanks for catching it.

Reply:
Thanks for the insight, just re-added the validation